infrastructure/code/code.go

```diff
@@ -194,7 +192,7 @@ func (sc *Scanner) Scan(ctx context.Context, path string, folderPath string) (is
 
 	var results []snyk.Issue
 	if sc.useIgnoresFlow() {
-		results, err = sc.UploadAndAnalyzeWithIgnores(span.Context(), folderPath)
+		results, err = sc.UploadAndAnalyzeWithIgnores(span.Context(), folderPath, files, startTime, changedFiles)
```

Comment:
This startTime parameter... can we implement this differently? I don't think we should have it in the signature. Also, looking at the code, it is wrong to exclude collecting the files, IMHO.

Reply:
I've had another look at this and we could keep a similar signature to the existing `UploadAndAnalyze` function, so we don't pass the startTime as a parameter. 

In terms of us excluding collecting the files. I agree but the current code excludes them and I'd like to use these metrics to compare how the two flows perform so I don't want to change how they work

Reply:
Actually, now that we've removed the Amplitude analytics this isn't even needed anymore